test/e2e/specs/test.js

```diff
@@ -8,12 +8,77 @@ module.exports = {
   'default e2e tests': function (browser) {
     browser
     .url(browser.launchUrl)
-      .waitForElementVisible('#app', 5000)
-      .assert.elementPresent('.randPersonBtn')
+      .waitForElementVisible(ui.dev.app, 5000)
+      .assert.elementPresent(ui.dev.randomBtn.person)
+      .pause(5000)
+      .end()
+  },
+
+  'verify person record has seven table records': function (browser) {
```

Comment:
Finally realized why this was not working in the evaluation version of WebStorm. I needed to upgrade the language of JavaScript.